utils/splunk/splunk_telemetry_base.py

```diff
@@ -263,6 +323,12 @@ def load_status(self):
         if self.status is None:
             self.status = CheckData()
 
+    def update_token_memory(self, base_url, token):
```

Comment:
Duplicate code between here and splunk_topology/check.py. Can we move this to the helper?